discovery/api/v1/api.go

```diff
@@ -145,8 +136,17 @@ func (w *Wrapper) GetServiceActivation(ctx context.Context, request GetServiceAc
 	if err != nil {
 		return nil, err
 	}
-	return GetServiceActivation200JSONResponse{
+	response := GetServiceActivation200JSONResponse{
 		Activated: activated,
+		Status:    ServiceStatusActive,
 		Vp:        &presentations,
-	}, nil
+	}
+	if refreshError := w.Client.GetServiceRefreshError(ctx, request.ServiceID, request.SubjectID); refreshError != nil {
```

Comment:
You should only do this when not properly activated? Or can it be both activated (valid VPs), but have an error? (1 DID failed, 1 succeeded)

Reply:
You only have an error when all DIDs failed. The idea is that you always call this API from your administration to check for green √. Combinations that could occur:

- not activated (yellow)
- activated but error (red)
- activated, no error (green)